docs/user/tls-passthrough.md

```diff
@@ -108,6 +113,5 @@ kubectl delete secret/server-certs
 ```
 
 ## Next Steps
-Checkout the [Developer Guide](../../DEVELOPER.md) to get involved in the project.
 
-[kind]: https://kind.sigs.k8s.io/
+Checkout the [Developer Guide](../../DEVELOPER.md) to get involved in the project.
```

Comment:
This doc has moved to https://github.com/envoyproxy/gateway/blob/main/docs/dev/README.md.